community/neo4j-slf4j/src/main/java/org/neo4j/logging/slf4j/Slf4jLog.java

```diff
@@ -239,4 +239,9 @@ public void bulk( Consumer<Log> consumer )
             consumer.accept( this );
         }
     }
+
+    private String convertFormat(String format)
```

Comment:
The conversion? Because SLF4J and String.format uses different placeholder markers (%s vs {})